docs/jwst/associations/technote_sdp_workflow.rst

```diff
@@ -19,9 +19,9 @@ processed. Instead of creating a single monolithic pipeline, the workflow uses
 the associations to determine what pipeline should be executed and when to
 execute them. In the figure below, this wait-then-execute process is represented
 by the ``workflow trigger``. The workflow reads the contents of an association
-to determine what exposures, and possibly other files, are needed to continue
-processing. The workflow then waits until all exposures exist. At that point,
-the related calibration step is executed with the association as input.
+file to determine what exposures, and possibly other files, are needed to
+continue processing. The workflow then waits until all exposures exist. At that
+point, the related calibration step is executed with the association as input.
```

Comment:
do we really want to say "calibration *step*" here? We're really talking about a pipeline, as opposed to a step.

Reply:
✅